routes/spentRoutes.js

```diff
@@ -0,0 +1,38 @@
+var db = require("../models");
+
+module.exports = function(app) {
+  app.get("/api/spent/:userid/", function(req, res) {
+    db.Spent.findAll({
+      where: {
+        UserId: req.params.userid
+      }
+    }).then(function(dbSpent) {
+      res.send(dbSpent);
```

Comment:
Should this be res.json?

Reply:
theoretically either will send the info, but res.send does not convert undefined and null to something supported by json where as res.json does.